pkg/common/cns-lib/vsphere/virtualmachine_test.go

```diff
@@ -0,0 +1,204 @@
+package vsphere
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"testing"
+
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/vapi/rest"
+	_ "github.com/vmware/govmomi/vapi/simulator"
+	"github.com/vmware/govmomi/vapi/tags"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+const (
+	regionCategoryName string = "k8s-region"
+	regionTagName      string = "region-1"
+	zoneCategoryName   string = "k8s-zone"
+	zoneTagName        string = "zone-A"
+)
+
+// CreateNewCategory creates a new category using tag manager
+func CreateNewCategory(ctx context.Context, categoryName string, cardinality string,
```

Comment:
Can we move unit tests util functions to "pkg/common/cns-lib/vsphere/virtualmachine_test_util.go" file

Reply:
Moved util functions to tagmanager_test_utils.go since these are generic util functions related to tag manager.